src/Symfony/Component/Form/Test/TestCaseSetUpTearDownTrait.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Test;
+
+use PHPUnit\Framework\TestCase;
+
+// Auto-adapt to PHPUnit 8 that added a `void` return-type to the setUp/tearDown methods
+
+if (method_exists(\ReflectionMethod::class, 'hasReturnType') && (new \ReflectionMethod(TestCase::class, 'tearDown'))->hasReturnType()) {
+    eval('
+    namespace Symfony\Component\Form\Test;
+
+    /**
+     * @internal
+     */
+    trait TestCaseSetUpTearDownTrait
+    {
+        protected function doSetUp(): void
```

Comment:
should be private so that the trait is local to the class that uses it and doesn't conflict with children (same on doTearDown)